KIC customization (#1279)
Kong/developer.konghq.com: app/kubernetes-ingress-controller/customize-image.md

---
title: "Customize Images"

description: |
  Customize the image name used for {{ site.kic_product_name }} and {{ site.base_gateway }}

> 

Reviewer: ```suggestion

breadcrumbs:
  - /kubernetes-ingress-controller/
  
```

content_type: reference
layout: reference

products:
  - kic

works_on:
  - on-prem
  - konnect
---

> {{ site.kic_product_name }} supports both {{ site.base_gateway }} OSS and Enterprise. If you install {{ site.kic_product_name }} using the default Helm charts, {{ site.base_gateway }} OSS will be installed.

Reviewer: Is this fine to say still, or should we not be mentioning OSS?

Author: I checked with Jessup. Let's leave this for now

> You can customize the products used by setting `gateway.image` and `controller.image` in your `values.yaml`. Once this is done, upgrade your Helm deployment.

Reviewer: ```suggestion
You can customize the products used by setting `gateway.image` and `controller.image` in your `values.yaml`, then upgrading your Helm deployment:
```

```bash

> helm upgrade <release_name> kong/ingress -n <namespace> --values ./values.yaml

Reviewer: ```suggestion
helm upgrade {release_name} kong/ingress -n {namespace} --values ./values.yaml
```

```

You can check which images are being used with the following query:

```bash
kubectl get pods -n kong -o jsonpath="{.items[*].spec.containers[*].image}"
```

> ## Common Images

Reviewer: ```suggestion
## Common images
```

These are some of the commonly used `values.yaml` files.

### {{ site.ee_product_name }}

You can switch to {{ site.ee_product_name }} using the `kong/kong-gateway` image:

```yaml
gateway:
  image:
    repository: kong/kong-gateway
    tag: {{ site.data.gateway_latest.release }}
```

> ### {{ site.kic_product_name }} Nightly

Reviewer: ```suggestion
### {{ site.kic_product_name }} nightly
```
